### PR TITLE
Added a new Data Science Course named Data Science Methodology in English Course list

### PR DESCRIPTION
**Pull request by manish-9245**
## What does this PR do?
Add resource(s)

### Description
It builds the methodology of learners to work with Data Science
### Why is this valuable (or not)?
It's a valuable resource for someone getting started with Data Science
### How do we know it's really free?
The Entire platform is free, you can click on the link and check it yourself.
### For book lists, is it a book? For course lists, is it a course? etc.
Yes it is a 3 hours course
## Checklist:
- [ 👍 ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [👍  ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [👍  ] Include author(s) and platform where appropriate.
- [ 👍 ] Put lists in alphabetical order, correct spacing.
- [ 👍 ] Add needed indications (PDF, access notes, under construction).
- [👍  ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -230,7 +230,7 @@
 ### Blockchain
 
 * [Blockchain Developer Tutorials](https://www.youtube.com/playlist?list=PLS5SEs8ZftgUTXs0OJD2LFpYBPr4L54id) - Gregory McCubbin (Dapp University)
-* [Blockchain Essentials](https://cognitiveclass.ai/courses/blockchain-course) - CognitiveClass.ai
+* [Blockchain Essentials](https://class.ai/courses/blockchain-course) - CognitiveClass.ai
```
Comment (eshellman) [inline courses/free-courses-en.md:233]: bad link

Comment (manish-9245) [reply courses/free-courses-en.md]: Fixed the bad link @eshellman

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by manish-9245** [comment]
Thank you!!

**Pull request merged**